src/components/PolicyFilter.js

```diff
@@ -269,4 +270,4 @@ class PolicyFilter extends Component {
     }
 }
 
-export default withModulesManager(injectIntl((withTheme(withStyles(styles)(PolicyFilter)))));
\ No newline at end of file
+export default withModulesManager(injectIntl((withTheme(withStyles(styles)(PolicyFilter)))));
```

Comment:
I tried to add extra line before but it does not catch it when pushed to the repo